packaging/vm-deployment/README.md

```diff
@@ -0,0 +1,29 @@
+# Developer VM Provisioning
+
+As a developer, if you would like to deploy a VM with BPFD running this will spin up the latest Fedora release with BPFD installed and running.
+
+### Pre-requisite
+
+- Virtualbox - The vagrant file uses virtualbox for cross-platform support. See [VirtualBox Downloads](https://www.virtualbox.org/wiki/Downloads) and choose your target OS.
+- Vagrant - Vagrant deploys the OS and will trigger the Anisble playbook. See [Vagrant Downloads](https://www.vagrantup.com/docs/installation) and choose your target OS.
+- Ansible - Once provisioned, Ansible will configure the VM. See [Installing Ansible](https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html) and choose your target OS.
+
+### Deploy the VM
+
+Once the dependencies are installed, simply clone bpfd and run the following commands:
+
+```console
+# Clone the bpfd repo:
+$ git clone https://github.com/nerdalert/bpfd.git
```

Comment:
Did you intend to leave `nerdalert` here? Should this be `redhat-et`?

Reply:
@nerdalert Can you fix this RQ :)

Reply:
lol ty